src/hotspot/share/classfile/bytecodeAssembler.cpp

```diff
@@ -89,6 +92,8 @@ ConstantPool* BytecodeConstantPool::create_constant_pool(TRAPS) const {
       default:
         ShouldNotReachHere();
     }
+PRAGMA_DIAG_POP
```

Comment:
One of these is another of the symbol_at_put cases that I don't understand.  Are there other cases in the switch that warn?  And if so, which ones and how?  There are no details of this one in the PR cover description.